trajopt_optimizers/trajopt_sqp/test/test_suite_utils.hpp

```diff
@@ -0,0 +1,31 @@
+#include <trajopt_utils/macros.h>
```

Comment:
I just realized we are missing licenses on a bunch of files in the new trajopt code. Looks like mostly in the unit tests

Reply:
Done.